engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.routes.js

```diff
@@ -9,13 +9,12 @@
  */
 angular.module('Bastion.errata').config(['$stateProvider', function ($stateProvider) {
     $stateProvider.state('errata', {
-        url: '/errata',
         abstract: true,
         controller: 'ErrataController',
         templateUrl: 'errata/views/errata.html'
     })
     .state('errata.index', {
-        url: '?repositoryId',
+        url: '/errata',
```

Comment:
The `?repositoryId` needs to be retained here because it is specifying a repositoryId query string parameter for the errata index state.  Perhaps change it to `url: /errata?repositoryId`?